packages/images/src/service.js

```diff
@@ -122,22 +122,23 @@ export class ImageService {
         suffix = `-${hash}`;
       }
 
-      let transformedFilename = nodePath.join(this.transformsPath, `${id}${suffix}.${format}`);
-
-      if (!(await fs.exists(transformedFilename))) {
-        try {
-          const sharpImage = sharp(originalPath);
-          if (resizeOptions) {
-            await sharpImage.resize(resizeOptions);
-          }
-          await sharpImage.toFile(transformedFilename);
-        } catch (err) {
-          console.error(err);
-          return res.status(500).send({ error: 'Internal server error' });
-        }
+      let transformedFilename = nodePath.join(this.transformsPath, `${id}${suffix}${format}`);
+
+      // If a cached copy of the transformed file exists, don't re-create it
```

Comment:
So... I'm pretty sure it introduces an error.

The previous logic was:

- if the cached image doesn't exist, generate it
- either way, send the image from the cache

Unless I'm misinterpreting the new code here, if a new image is created, it won't be sent

Reply:
Oh also, please check against the latest commits, there's a merge conflict (I think we both may have made similar changes concurrently based on your review of my PR, but this PR does include improvements I didn't do so would be great to merge it)

Reply:
Good catch -- not sure how I missed that in the first place 😭